Ghidra/Features/Base/src/main/java/ghidra/app/util/bin/format/pef/LoaderInfoHeader.java

```diff
@@ -27,11 +27,11 @@
  * See Apple's -- PEFBinaryFormat.h
  * <pre>
  * struct PEFLoaderInfoHeader {
- *     SInt32  mainSection;              // Section containing the main symbol, -1 => none.
```

Comment:
You could also just not use &amp;gt here to simplify things.  A == or : might convey the same meaning.

Reply:
I added an `{@literal}` surrounding the struct. Within the `<pre>` it keeps the formatting and automatically escapes as necessary.